hazelcast/src/test/java/com/hazelcast/internal/ascii/RestTest.java

```diff
@@ -230,6 +226,8 @@ public void addWanConfig() throws Exception {
 
     @Test
     public void updatePermissions() throws Exception {
+        Config config = new Config();
+        setup(config);
```

Comment:
Minor: having two calls `setup(null)` and `setup(config)` is confusing. If you need the config object after the setup why don't we do
```
private Config setup() {
    Config config = new Config();
    ...
    return config;
}